community/kernel/src/main/java/org/neo4j/kernel/api/proc/FieldSignature.java

```diff
@@ -71,6 +98,12 @@ private FieldSignature( String name, Neo4jTypes.AnyType type, Neo4jValue default
         }
     }
 
+    /** Fields that are not supported full stack (ie. by Cypher) need to be mapped from Cypher to internal types */
+    public Object map( Object input )
```

Comment:
Are we ever using this default method, it looks like we are always overriding. Maybe make it abstract?

Reply:
We only override it when there are byte[] input types , and only for that specific field, otherwise the default behaviour is used.